DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/mvdr_beamforming/src/Transpose.hpp

```diff
@@ -45,12 +45,7 @@ event SubmitTransposeKernel(queue& q) {
                 "k_num_cols_in must be evenly divisible by k_pipe_width");
 
   return q.submit([&](handler& h) {
-    h.single_task<TransposeKernelName>([=]() {
-      // start the transposer
-      Transposer<T, k_num_cols_in, k_pipe_width, MatrixInPipe, MatrixOutPipe>
-          TheTransposer;
-      TheTransposer();
-    });
+    h.single_task<TransposeKernelName>(Transposer<T, k_num_cols_in, k_pipe_width, MatrixInPipe, MatrixOutPipe>);
```

Comment:
I think we can simplify this to:
```c++
q.single_task<TransposeKernelName>(Transposer<T, k_num_cols_in, k_pipe_width, MatrixInPipe, MatrixOutPipe>{});
```

Reply:
i.e. you don't need both submit and single_task like your current version does. Just q.sinlge_task is sufficient.